tests/modules/teams/conftest.py

```diff
@@ -24,3 +24,20 @@ def team_for_regular_user(db, regular_user, readonly_user):
     db.session.delete(regular_user_team_member)
     db.session.delete(team)
     db.session.commit()
+
+@pytest.yield_fixture()
+def team_for_nobody(db):  # pylint: disable=invalid-name
```

Comment:
I also wonder if it supports nesting, e.g. if we just put a `tests/.pylintrc` with a few lines of config instead of a complete copy...